src/pages/Auth/Login.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import "./Auth.css";
+import { useAuth } from "../../context";
+import { useNavigate, Link } from "react-router-dom";
+import axios from "axios";
+import { Input } from "../../components";
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [togglePassword, setTogglePassword] = useState(false);
+  const [error, setError] = useState("");
+  const { setUser } = useAuth();
+  const navigate = useNavigate();
+
+  const loginHandler = async (e) => {
+    e.preventDefault();
+    try {
+      const { data } = await axios.post("/api/auth/login", { email, password });
+      const { foundUser, encodedToken } = data;
+      setUser(foundUser);
+      localStorage.setItem("token", encodedToken);
+      navigate("/home");
+    } catch (error) {
+      console.error(error);
```

Comment:
console can be removed as you have a setter of error

Reply:
yes. but console added from dev perspective if anything went wrong we should know the errors